### PR TITLE
Feature/update build

### PR DESCRIPTION
**Pull request by modulo11**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -21,9 +21,6 @@
 	</description>
 
 	<properties>
-		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-		<project.resources.sourceEncoding>UTF-8</project.resources.sourceEncoding>
```
Comment (beatngu13) [inline pom.xml]: Seems to be not set by parent?

File: pom.xml [hunk 2]
```diff
@@ -9,9 +9,9 @@
 		<relativePath />
 	</parent>
 
-	<groupId>org.retest</groupId>
+	<groupId>de.retest.services</groupId>
 	<artifactId>rebazer</artifactId>
-	<version>0.9.4-SNAPSHOT</version>
+	<version>0.10-SNAPSHOT</version>
```
Comment (beatngu13) [inline pom.xml]: Intended to have no patch version anymore?

File: .gitignore [hunk 3]
```diff
@@ -1,25 +1,69 @@
...
+
+##########################
+## VS Code
+##########################
+.vscode/
+
+##########################
+## OS X
+##########################
+.DS_Store
```
[the remark this answers is not in the document]
Comment (modulo11) [reply .gitignore]: This is mainly a sync between all service projects. But I agree.

File: pom.xml (hunk 2 above)
Comment (modulo11) [reply pom.xml]: Yes, this also comes from the sync.

File: pom.xml (hunk 1 above)
Comment (modulo11) [reply pom.xml]: Not sure about this actually :D

**Pull request merged**